pkg/idtools/idtools_unix.go

```diff
@@ -26,14 +26,25 @@ func mkdirAs(path string, mode os.FileMode, ownerUID, ownerGID int, mkAll, chown
 	// so that we can chown all of them properly at the end.  If chownExisting is false, we won't
 	// chown the full directory path if it exists
 	var paths []string
```

Comment:
`paths` isn't used until line 47 below, so this should be moved to immediately before it's used.